hello-world-api/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "hello-wolrd-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node dist/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc",
+    "dev": "nodemon dist/index.js",
+    "lint": "biome check .",
+    "format": "biome format ."
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^5.1.0"
+  },
+  "devDependencies": {
+    "@biomejs/biome": "1.9.4",
+    "@types/express": "^5.0.1",
+    "@types/node": "^20.0.0",
+    "nodemon": "^3.1.10",
+    "typescript": "^5.0.0"
```

```suggestion
    "@types/express": "5.0.1",
    "@types/node": "20.0.0",
    "nodemon": "3.1.10",
    "typescript": "5.0.0"
```
Eh sempre bom tirar os ^ pra fixar as versões e não rolar auto update
Do `express` ali nas `dependencies` tbm

---

Ok! Feito